sorc/ncep_post.fd/MISCLN.f

```diff
@@ -2910,7 +2938,8 @@ SUBROUTINE MISCLN
                EGRID1(I,J) = LOG(PMID(I,J,LM)/EGRID2(I,J))   &
                            / LOG(PMID(I,J,LM)/PMID(I,J,LM-1))
 
-        IF (MODELNAME == 'GFS' .OR. MODELNAME == 'FV3R') THEN
+        IF (MODELNAME == 'GFS' .OR. MODELNAME == 'FV3R' &
+           .OR. MODELNAME == 'FV3R') THEN
```

Comment:
@EdwardColon-NOAA The model name "FV3R" is duplicated here.

Reply:
This duplication was resolved.